src/Microsoft.Identity.Web/Constants/LogMessages.cs

```diff
@@ -3,25 +3,25 @@
 
 namespace Microsoft.Identity.Web
 {
-    /// <summary>
-    /// Constants related to the log messages.
-    /// </summary>
-    internal static class LogMessages
-    {
-        public const string MissingRoles = "The 'roles' or 'role' claim does not contain roles '{0}' or was not found";
-        public const string MissingScopes = "The 'scope' or 'scp' claim does not contain scopes '{0}' or was not found";
-        public const string ExceptionOccurredWhenAddingAnAccountToTheCacheFromAuthCode = "Exception occurred while adding an account to the cache from the auth code. ";
+	/// <summary>
+	/// Constants related to the log messages.
+	/// </summary>
+	internal static class LogMessages
+	{
+		public const string MissingRoles = "The 'roles' or 'role' claim does not contain roles '{0}' or was not found";
+		public const string MissingScopes = "The 'scope' or 'scp' claim does not contain scopes '{0}' or was not found";
+		public const string ExceptionOccurredWhenAddingAnAccountToTheCacheFromAuthCode = "Exception occurred while adding an account to the cache from the auth code. ";
 
-        // Diagnostics
-        public const string MethodBegin = "Begin {0}. ";
-        public const string MethodEnd = "End {0}. ";
+		// Diagnostics
+		public const string MethodBegin = "Begin {0}. ";
+		public const string MethodEnd = "End {0}. ";
 
-        // Caching
-        public const string DeserializingSessionCache = "Deserializing session {0}, cache key {1}. ";
-        public const string SessionCacheKeyNotFound = "Cache key {0} not found in session {1}. ";
-        public const string SerializingSessionCache = "Serializing session {0}, cache key {1}. ";
-        public const string ClearingSessionCache = "Clearing session {0}, cache key {1}. ";
+		// Caching
+		public const string DeserializingSessionCache = "Deserializing session {0}, cache key {1}. ";
```

Comment:
Please consider using LoggerMessage as described at https://docs.microsoft.com/en-us/dotnet/core/extensions/high-performance-logging instead of doing costly message formats into strings.